src/store.tsx

```diff
@@ -31,3 +31,5 @@ export const store = createStore<StoreState>(
   initialState,
   devToolsEnhancer({}),
 );
+
+export const ABC = true;
```

Comment:
Why this line? and why the `.tsx` (I try to only put `tsx` if the file contain some jsx inside :wink: )